### DISCUSSION TITLE
Add network statistics to app

### DISCUSSION THREAD

**Discussion by Mhowser** [mullvad/mullvadvpn-app#8944] [Feature requests & Ideas]
## Feature Request
Add network statistics to app
Operating system: All

App version: N/A
### Feature description

A previous VPN client I used to use had a feature were users could quickly view real-time TX and RX speeds in Mb/s, including total megabytes/gigabytes for the current session. In Android, this information is displayed in the notification card of the VPN. This is also displayed in the desktop client as well. 

I think this is quite useful to have and would love to see it in this great app.

**Comment by inson1** [comment]
I would love that

**Comment by MrChocolatine** [comment]
+1
